ruby-challenges.rb

```diff
@@ -9,26 +9,84 @@
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
+#create a method that takes in an array as an argument.
+#iterate through array.
 
-
+only_contains_o = beverages_array.select do |value, string|
+    new_value = []
+    if value.include? letter_o
+        p new_value << value
+    end
+end
+only_contains_t = beverages_array.select do |value, string|
+    new_value = []
+    if value.include? letter_t
+        p new_value << value
+    end
+end
```

Comment:
These could be refactored into methods-- if You find you have some time I recommend you do. It will make your life a lot easier to know how to make methods of on a regular basis.